code/style.css

```diff
@@ -1,4 +1,189 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+/* Header */
+
+header {
+  background: wheat;
+  height: 150px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+header img {
+  float: none;
+  
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  color: black;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-decoration: none;
+  font-size: 20px;
+}
+
+.hamburger {
+  display: flex;
+  width: 100px;
+  height: 50px;
+  flex-direction: column;
+  justify-content: space-between;
+  
+}
+
+.hamburger div {
+  width: 50px;
+  height: 10px;
+  background: black;
+  border-radius: 5px;
+}
+
+/* body */
+
+body {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  background: wheat;
+  
+}
+
+/* introduction section */
+
+.introduction {
+  background: darkolivegreen;
+  border: solid black;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-align: center;
+
+}
+
+/* Main news */
+
+.big-news {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.big-news article {
+  margin: 5px;
+  align-content: space-between;
+  justify-content: center;
+  border: 1px solid;
+  padding: 2px;
+  box-shadow: 2px 2px;
+}
+
+.big-news img {
+  width: 90%;
+  
+}
+
+div:hover {
+  background-color: whitesmoke;
+  opacity: 0.5;
+}
+
+/* Other news grid */
+
+.other-news {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.other-news article {
+  margin: 10px;
+  align-content: space-between;
+  justify-content: center;
+  border: 1px solid;
+  padding: 2px;
+  box-shadow: 2px 2px;
+}
+
+.other-news img {
+  width: 90%;
+}
+
+/* footer */
+
+footer {
+  height: 50px;
+  width: 100%;
+  border: solid black;
+  background: darkolivegreen;
+
+}
+
+footer h2 {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-align: center;
+  
+}
+
+/* media tablet*/
+
+@media (min-width: 667px) and (max-width: 1024px) {
+
+  .other-news {
+    flex-direction: row;
+    flex-wrap: wrap;
+  
+  }
+
+  .other-news article {
+   width: 40%;
+   height: 50%;
+   
+  
+  }
+
+  header {
```

Comment:
Same here, try to keep what's on top of your page in order but I'm nitpicking on details since you're doing good 😎